src/assets/scss/abstracts/_variables.scss

```diff
@@ -1 +1,10 @@
-$BG_COLOR: red;
+$color-white: #fff;
+$color-green: green;
```

Comment:
You can separate the colors into a `colors.scss` and keep this file for font sizes and other things, but overall this is what we are looking for